examples/astro/src/pages/web-components.astro

```diff
@@ -231,6 +231,38 @@ import Layout from '../layouts/Layout.astro';
             );
         });
       </script>
+      <h2 class="ds-text-heading--2xl">Review example</h2>
+      <ds-review heading="Heading for the Review Component">
```

Comment:
This example is working great, but it looks a little squished. Maybe we should rearrange the example page a bit so the review component isn't inside a `measure--base`

Reply:
@pwolfert Thanks for catching this! Moved the review example outside of the `measure--base`, so it should be un-squished now. :)